sw/otbn/crypto/boot.s

```diff
@@ -157,6 +157,18 @@ attestation_keygen:
   la        x22, y
   bn.sid    x2, 0(x22)
 
+  /* Compute both sides of the Weierstrauss equation.
```

Comment:
nice. can we add a comment that this PR also adds the affine check to key gen from boot services please?